### DISCUSSION TITLE
Is it possible to add a note to the documentation to help newcomers?

### DISCUSSION THREAD

**Discussion by Pete-Taylor** [mkdocstrings/mkdocstrings#352] [General]
Hi there,

It's a minor gripe, but as a newcomer to mkdocs, mkdocstrings and YAML, I had some trouble when attempting to change the `docstring_style` to ReStructuredText. I came across [this issue](https://github.com/mkdocstrings/mkdocstrings/issues/278), which appeared to give the solution, but when I manually added the options to my `mkdocs.yml` file, I was faced with:
```
$ mkdocs serve
INFO     -  Building documentation...
ERROR    -  Config value: 'plugins'. Error: Invalid Plugins configuration
Aborted with 1 Configuration Errors!
```
It took me quite some time of experimenting and searching before I came across [this comment](https://github.com/comwes/mkpdfs-mkdocs-plugin/issues/5#issuecomment-570801791) by "cfont" (on a different plugin), which left me feeling a bit embarrassed for not noticing the extra 2 spaces of indentation right away!

It would be good if a note could be added to [this section](https://mkdocstrings.github.io/handlers/python/#selection) of the documentation saying something along the lines of "please note the extra indentation for the mkdocstrings options", or something along those lines (just for those of us who don't copy and paste).

**Comment by pawamoy** [comment]
Hi @Pete-Taylor, thank you for your feedback. However I'm not sure mkdocstrings would be the right place to add such a note. The choice to use YAML comes from MkDocs. And indentation is the basis of YAML documents, so users should of course try and see how YAML is written. Well... I guess a link to YAML's syntax somewhere in the docs could'nt hurt, though I'm not sure where :confused: You mention a specific section, but there's already an example right at its end, with a working indentation. Let me know what you think.

**Reply by pawamoy** [reply]
OK we can probably add a reminder with a link to this https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html (and optionally this https://yaml-multiline.info/) in this section https://mkdocstrings.github.io/usage/#autodoc-syntax

**Reply by Pete-Taylor** [reply]
Hi @pawamoy,

What happened to me, when manually typing in the example, was that I overlooked the extra indentation in the example. After entering `- mkdocstrings:`, I pressed 'Enter' followed by 'Tab' and VSCode automatically indented by 2 characters more than the previous line, so my YAML looked like:

```
plugins:
  - mkdocstrings:
    handlers:
      python:
        selection:
          filters:
```

whereas it needed to look like:

```
plugins:
  - mkdocstrings:
      handlers:
        python:
          selection:
            filters:
```

The Ansible document still didn't really explain why `handlers:` needs to be indented six spaces, when the previous `- mkdocstrings:` item was only indented by 2 spaces.

I did, however, find a document that explains the idiosyncrasy quite nicely: https://docs.saltproject.io/en/3000/topics/troubleshooting/yaml_idiosyncrasies.html#nested-dictionaries

tldr; You are right; the user should learn YAML. However, I feel that a little note on this idiosyncrasy would probably still be helpful.
